<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StylesManager</class>
 <widget class="QDialog" name="StylesManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="stylesTypes">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="paragraphStylesTab">
      <attribute name="title">
       <string>Paragraph Styles</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeView" name="paragraphStylesListView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="newParagraphStyleButton">
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveParagraphStyleButton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteParagraphStyleButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="paragraphOptionsTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="StylesGeneralTab" name="paragraphGeneralTab">
          <attribute name="title">
           <string>General</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="characterStylesTab">
      <attribute name="title">
       <string>Character Styles</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTreeView" name="characterStylesListView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="newCharacterStyleButton">
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveCharacterStyleButton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteCharacterStyleButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="characterOptionsTabs">
         <widget class="StylesGeneralTab" name="characterGeneralTab">
          <attribute name="title">
           <string>General</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="stylePreview" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyToCursorButton">
       <property name="text">
        <string>Apply Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitWithoutApplyButton">
       <property name="text">
        <string>Exit without applying</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StylesGeneralTab</class>
   <extends>QWidget</extends>
   <header>dialogs/StylesGeneralTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KCategorizedView</class>
   <extends>QWidget</extends>
   <header>kcategorizedview.h</header>
   <container>1</container>
  </customwidget>
</customwidgets>
 <resources/>
 <connections/>
</ui>
